docs/presentation/RepeatableRelationshipPathSpecification.md

```diff
@@ -0,0 +1,88 @@
+# Repeatable Relationship Path Specification
+
+> TypeScript type: [RepeatableRelationshipPathSpecification]($presentation-common).
+
+Repeatable relationship path specification is used to define a relationship path to an ECClass, optionally traversing through the same relationship
+multiple times or recursively.
```

Comment:
```suggestion
This specification allows users to specify an indirect relationship between ECInstances. Indirect relationship is expressed as a chain of relationship classes which can optionally be traversed recursively.
```

Reply:
Can you explain why you suggest this specification is to specify an "**indirect** relationship"?

Reply:
I posted an alternative suggestion, I would like to have your feedback on it